client/extensions/woocommerce/state/sites/coupons/handlers.js

```diff
@@ -1,31 +1,39 @@
 /**
+ * External dependencies
+ */
+import { trim } from 'lodash';
+
+/**
  * Internal dependencies
  */
 import debugFactory from 'debug';
 import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
 import request from 'woocommerce/state/sites/http-request';
 import {
-	WOOCOMMERCE_COUPONS_REQUEST_PAGE,
+	WOOCOMMERCE_COUPONS_REQUEST,
 } from 'woocommerce/state/action-types';
 import { couponsPageUpdated } from './actions';
 
 const debug = debugFactory( 'woocommerce:coupons' );
 
 export default {
-	[ WOOCOMMERCE_COUPONS_REQUEST_PAGE ]: [
+	[ WOOCOMMERCE_COUPONS_REQUEST ]: [
 		dispatchRequest( requestCouponsPage, requestCouponsPageSuccess, apiError )
 	]
 };
 
 export function requestCouponsPage( { dispatch }, action ) {
-	const { siteId, pageIndex, perPage } = action;
-	const path = `coupons?page=${ pageIndex }&per_page=${ perPage }`;
+	const { siteId, params } = action;
```

Comment:
Again should the method names here be updated to remove `Page` since it now supports all params?

Reply:
Yes they should. 👍

Reply:
Addressed in b37e30c